### DISCUSSION TITLE
Adding non listed OS

### DISCUSSION THREAD

**Discussion by hjbedoya** [procount/pinn#826] [Q&A] [answered]
Hi,  is it's possible to install non listed OS like Dosbian or CoCoPi? Or is there a way to add those OS to the list? I got the image files but don't know how to add/install.
I'm trying to do this on a Raspberry Pi 5 with a 500gb SSD. 

Thnks

**Comment by procount** [comment, marked as answer]
If you want to add other OSes, you need to convert them to the correct format first, and add additional meta-files to describe how PINN should install them. There are instructions in the main documentation and on the wiki. Please see: https://github.com/procount/pinn/blob/master/README_PINN.md#how-to-create-a-custom-os-version and https://github.com/procount/pinn/wiki/How-to-Create-a-Multi-Boot-SD-card-out-of-2-existing-OSes-using-PINN.
However, these are just general instructions as each OS will have its own nuances and requirements for conversion to multi-boot.

**Reply by hjbedoya** [reply]
Hi, Thanks for your answer so I need to copy the OS that i want to install to the OS folder and modify some files. the OS that i copy must be the installed version?

**Reply by procount** [reply]
Please read the linked instructions. It is quite an advanced activity and you need to understand a fair bit to do it. 
Also check that your intended OSes are suitable for the Pi5 before you start. You will need lots of disk space to convert an OS.
